app/services/proceeding_type_full_text_search.rb

```diff
@@ -0,0 +1,68 @@
+# This class is responsible for searching the proceeding types table for matching terms.
+#
+# It uses Postresql's Full Text Search (see https://www.postgresql.org/docs/9.5/textsearch-intro.html)
+#
+# Results are currently retuned by ranked by relevance, i.e. search terms which appear in
```

Comment:
spelling error here, assume this should be 'returned and ranked by'

Reply:
Copy paste from Apply!